capstone/src/main/java/com/google/sps/servlets/Login.java

```diff
@@ -23,7 +23,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       redirectUrl = userService.createLoginURL("/login");
     } else {
       logMsg = "user is already logged in";
-      redirectUrl = "statistics.html";
+      //redirectUrl = "statistics.html";
```

Comment:
How will users reach the stats page?

Reply:
showStats();

Reply:
I've updated my CheckUserStatus and Login servlets accordingly